<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrbitElementsInfo</class>
 <widget class="QDockWidget" name="OrbitElementsInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>101</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>101</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
   <set>QDockWidget::DockWidgetMovable</set>
  </property>
  <property name="windowTitle">
   <string>Orbit Elements</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>398</width>
         <height>412</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QGroupBox" name="VerboseGroupNew">
          <property name="title">
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="a1Label">
             <property name="text">
              <string> a </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="a2Label">
             <property name="text">
              <string> a [km]</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="a3Label">
             <property name="text">
              <string>km</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="e1Label">
             <property name="text">
              <string>e</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="e2Label">
             <property name="text">
              <string> e </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="i1Label">
             <property name="text">
              <string>i</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="i2Label">
             <property name="text">
              <string>i</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="i3Label">
             <property name="text">
              <string>rad</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="an1Label">
             <property name="text">
              <string>Asc node </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="an2Label">
             <property name="text">
              <string>Omega</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="an3Label">
             <property name="text">
              <string>hrs</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="p1Label">
             <property name="text">
              <string>Periapsis arg</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="p2Label">
             <property name="text">
              <string>w [rad] </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="p3Label">
             <property name="text">
              <string>rad</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="T01Label">
             <property name="text">
              <string>M&lt;sub&gt;0&lt;/sub&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="T02Label">
             <property name="text">
              <string>M0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="T03Label">
             <property name="text">
              <string>rad</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="blank1Label">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="tA1Label">
             <property name="text">
              <string> True Anomaly </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="tA2Label">
             <property name="text">
              <string>theta [rad] </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLabel" name="tA3Label">
             <property name="text">
              <string>rad</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="mA1Label">
             <property name="text">
              <string>M</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLabel" name="mA2Label">
             <property name="text">
              <string>M </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="eA1Label">
             <property name="text">
              <string>E</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLabel" name="eA2Label">
             <property name="text">
              <string>E [rad] </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="9" column="2">
            <widget class="QLabel" name="eA3Label">
             <property name="text">
              <string>rad </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="pa1Label">
             <property name="text">
              <string>r&lt;sub&gt;P&lt;/sub&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QLabel" name="pa2Label">
             <property name="text">
              <string>rp [km] </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="10" column="2">
            <widget class="QLabel" name="pa2Label">
             <property name="text">
              <string>km</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="aa1Label">
             <property name="text">
              <string>r&lt;sub&gt;A&lt;/sub&gt;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QLabel" name="aa2Label">
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="11" column="2">
            <widget class="QLabel" name="aa3Label">
             <property name="text">
              <string>km</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="sma1Label">
             <property name="text">
              <string>b</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QLabel" name="sma2Label">
             <property name="text">
              <string>b [km] </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight</set>
             </property>
            </widget>
           </item>
           <item row="12" column="2">
            <widget class="QLabel" name="sma2Label">
             <property name="text">
              <string>km </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>

        <item>
         <widget class="QGroupBox" name="GraphicGroup">
          <property name="title">
           <string>OE Graphic</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">

          </layout>
         </widget>
        </item>

        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionVerbose_DisplayNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OE Verbose</string>
   </property>
  </action>
  <action name="actionGraphic_Display">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OE Graphic</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionGraphic_Display</sender>
   <signal>toggled(bool)</signal>
   <receiver>GraphicGroup</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVerbose_DisplayNew</sender>
   <signal>toggled(bool)</signal>
   <receiver>VerboseGroupNew</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
